docs/setup/setup.md

```diff
@@ -16,6 +16,21 @@ $ clara pull dicom
 
 Next, we'll configure Clara DICOM Adapter.
 
+## Configure Storage Mount
+
+DICOM Adapter, by default, stores all received DICOM instances temporarily under `/clara-io/clara-core/payloads` which is mounted
+to `/payloads` inside the DICOM Adapter container.
+
+Disk storage size required for the temporary space depends on many factors: size per job, time to process each pipeline per job and the total number of 
+jobs per day, etc.  It is advised to allocate enough space for the temporary space or perhaps mount `/clara-io` on an network attached
+storage (NAS) devices.
```

Comment:
Is it possible to add instructions on what to change to migrate the `/clara-io` location?

Reply:
Updated.